src/Sylius/Bundle/WebBundle/Behat/WebContext.php

```
@@ -671,4 +691,14 @@ public function iGoToPageForProductWithEmptySlug()
     {
         $this->visitPath('/p/');
     }
-}
\ No newline at end of file
+
+    private function assertRoute($route)
+    {
+        $this->assertSession()->addressEquals($this->generatePageUrl($route));
+
+        try {
+            $this->assertStatusCodeEquals(200);
+        } catch (UnsupportedDriverActionException $e) {
+        }
```

Suppressing the error? Why?